### ISSUE TITLE
Skill recommandations

### ISSUE THREAD

**Issue by patou** [Zenika/skillZ#108]
When add a new skill, there are the big list of skill ordered displayed.
It's was great to see this list with a small selection of recommandations from others skills I allready filled.
If I add Java, the system can recommand to add spring or hibernate but not angular.
It's like product recommandations in Amazon.

**Comment by patou** [comment]
https://cloud.google.com/recommendations

**Comment by Ked57** [comment]
Yes ! We will need to develop some sort of recommendation system for the Slack bot, and it could be reused for in app recommendations for sure

**Comment by StephanieHP** [comment]
@MailyLehoux @benjaminbrgs a voir s'il est possible d'implémenter de l'IA ou des tags afin de recommander une compétence en fonctions de ce que les autres ont remplit (ex : fonctionnement site ecommerce)

**Comment by rguillome** [comment]
Recommandation based on collaborative filtering asked in the issue #626
